apps/web-staking/src/app/globals.css

```diff
@@ -60,6 +60,27 @@
     clip-path: polygon(15px 0, 100% 0, 100% calc(100% - 15px), calc(100% - 15px) 100%, 0 100%, 0 15px);
   }
 
+  .dropdown-options::-webkit-scrollbar {
+    width: 10px;
+    background: #000;
+  }
+
+  .dropdown-options::-webkit-scrollbar-thumb {
+    background: #1F1B1B;
+    border-right: 2px black solid;
+    border-top: 2px black solid;
+    border-bottom: 2px black solid;
+    background-clip: padding-box;
+  }
+
+  .dropdown-clip-path {
+    clip-path: polygon(0 0, 100% 0, 100% calc(100% - 8px), calc(100% - 8px) 100%, 0 100%, 0 8px);
+  }
+
+  .dropdown:hover + .dropdown-border {
+    background-color: #FF2C3A;
+  }
+
```

Comment:
Can this not be part of the component's stylings ? 
This adds a dependency for adding the global css to any app that would want to use that dropdown component.

Reply:
There are problems with the application of common styles while there is no monorepo, only in this way created styles are applied correctly

Reply:
Seems to be worth trying different things, having that dependency is not good. Please make sure its documented in the component documentation !

Reply:
I tried to apply styles directly in components, but as a result I got errors and incorrect styles